src/cuga/backend/cuga_graph/nodes/cuga_lite/tool_calling.py

```
@@ -0,0 +1,125 @@
+            cfg["cuga_lite_bind_tools_apps"] = list(tc.apps)
+        else:
+            cfg["cuga_lite_bind_tools_mode"] = "all"
+        # Always pin the boolean for an explicit native/hybrid selection so a
+        # global setting / profile can't widen it by enabling find_tools.
+        cfg["cuga_lite_bind_tools_include_find_tools"] = bool(tc.include_find_tools)
+        if tc.max_bound_tools is not None:
+            cfg["cuga_lite_bind_tools_max_count"] = int(tc.max_bound_tools)
+        if tc.tool_choice is not None:
+            cfg["cuga_lite_tool_choice"] = tc.tool_choice
+        return cfg
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)